src/FormHandler/CommentFormHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace BiffBangPow\MessageBoard\FormHandler;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Component\HttpFoundation\Request;
+use BiffBangPow\MessageBoard\Model\Comment;
+
+class CommentFormHandler
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+    /**
+     * @var EntityRepository
+     */
+    private $threadRepository;
+
+    /**
+     * CommentFormHandler constructor.
+     * @param EntityManagerInterface $entityManager
+     * @param EntityRepository $threadRepository
+     */
+    public function __construct(EntityManagerInterface $entityManager, EntityRepository $threadRepository)
+    {
+
+        $this->entityManager = $entityManager;
+        $this->threadRepository = $threadRepository;
+    }
+
+    /**
+     * @param Request $request
+     * @param int $id
+     */
+    public function handle(Request $request, int $id){
+        $comment = new Comment();
+        $thread = $this ->threadRepository->find($id);
+
+        $comment -> setContent((htmlspecialchars($request->get('content'))));
```

Comment:
I'd be very wary about changing an end users content. This has several legal complications and although thats not going to be a problem I would put whatever the user throws into the text area direct to the database then ensure your twig template handles the special characters.

Reply:
As I've just learned, this codes does nothing, bc with PHP 5.4 the default encoding was changed to UTF-8. I will remove it.